terraform/cloud-platform-components/resources/fluentd-es/templates/00-namespace.yaml

```diff
@@ -0,0 +1,13 @@
+#This file creates the logging namespace for flentd-es. It is named as is to ensure it's the first file deployed.
```

Comment:
We do not want to manage the namespace in the chart (I don't think any chart we're using does this).